docs/node/CATEGORIES/12-governance.md

```diff
@@ -112,10 +112,20 @@ Returns list of proposals.
 
 ```ts title="client.governance.listGovProposals()"
 interface governance {
-  listGovProposals ({
-    type = ListProposalsType.ALL,
-    status = ListProposalsStatus.ALL
-  } = {}): Promise<ProposalInfo[]>
+  listGovProposals (options: ListProposalsOption = {}): Promise<ProposalInfo[]>
+}
+
+interface ListProposalsOptions {
+  type?: ListProposalsType
+  status?: ListProposalsStatus
+  cycle?: number
+  pagination?: ListProposalsPagination
+}
+
+interface ListProposalsPagination {
+  start?: string
```

Comment:
noticed some discrepancy between the expected type for `start` between here and [ain](https://github.com/DeFiCh/ain/blob/master/src/masternodes/rpc_proposals.cpp#L803-L807), but seems like the wrong type is on `ain`'s side.
 cc: @shohamc1

Reply:
I have informed the blockchain team previously and they are awared about the type error.